core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.directive.ts

```diff
@@ -131,169 +147,136 @@ angular.module('oppia').directive('statisticsTab', [
-                      COMPLETION_RATE_PIE_CHART_OPTIONS);
-                    $scope.COMPLETION_RATE_PIE_CHART_OPTIONS1.title = title1;
-
-                    var title2 = 'Solution usage statistics';
-                    $scope.COMPLETION_RATE_PIE_CHART_OPTIONS2 = angular.copy(
-                      COMPLETION_RATE_PIE_CHART_OPTIONS);
-                    $scope.COMPLETION_RATE_PIE_CHART_OPTIONS2.title = title2;
 
                     $scope.stateName = stateName;
                     $scope.stateStats = stateStats;
                     $scope.improvementType = improvementType;
 
-                    var usefulFeedbackCount = (
+                    $scope.COMPLETION_RATE_PIE_CHART_OPTIONS1 = (
+                      newCompletionRatePieChartOptions(
+                        'Answer feedback statistics'));
+
+                    $scope.COMPLETION_RATE_PIE_CHART_OPTIONS2 = (
+                      newCompletionRatePieChartOptions(
+                        'Solution usage statistics'));
+
+                    const usefulFeedbackCount = (
                       $scope.stateStats.useful_feedback_count);
-                    var totalAnswersCount = (
+                    const totalAnswersCount = (
                       $scope.stateStats.total_answers_count);
+                    const numTimesSolutionViewed = (
+                      $scope.stateStats.num_times_solution_viewed);
+
                     if (totalAnswersCount > 0) {
                       $scope.hasExplorationBeenAnswered = true;
                     }
+
                     $scope.pieChartData1 = [
                       ['Type', 'Number'],
                       ['Default feedback',
                         totalAnswersCount - usefulFeedbackCount],
                       ['Specific feedback', usefulFeedbackCount],
                     ];
 
-                    var numTimesSolutionViewed = (
-                      $scope.stateStats.num_times_solution_viewed);
                     $scope.pieChartData2 = [
                       ['Type', 'Number'],
                       ['Solutions used to answer', numTimesSolutionViewed],
-                      ['Solutions not used', totalAnswersCount - (
-                        numTimesSolutionViewed)]
+                      ['Solutions not used',
+                        totalAnswersCount - numTimesSolutionViewed]
                     ];
 
-                    var _getVisualizationsHtml = function() {
-                      var htmlSnippets = visualizationsInfo.map(function(
-                          vizInfo) {
-                        var escapedData =
-                          HtmlEscaperService.objToEscapedJson(vizInfo.data);
-                        var escapedOptions =
-                          HtmlEscaperService.objToEscapedJson(vizInfo.options);
+                    $scope.visualizationsHtml = visualizationsInfo.map(info => {
+                      const vizElement = $(
```

Comment:
@marianazangrossi Flagging this as a use of jQuery that we want to get rid of.

@brianrodri  if we can do this using html and standard angular methods, that would be better. Is it possible? (Maybe in a separate PR.) We are trying to avoid manipulation of HTML in the JS files.

Reply:
Thanks for the catch @seanlip

Reply:
This will be fixed by #9605